docs/getting_started.md

```diff
@@ -0,0 +1,31 @@
+# 📚 Overview
+
+Dozer is a Real-Time SQL Engine that connects to any of your data sources, transforms and stores the data in an embedded cache.
+
+**Core Features**
+
+* Simple to Use
+Dozer instantly generates fully indexed gRPC and REST APIs automatically. All you need is to configure a YAML file with your data source configuration and the APIs you want to deploy. 
+
+* Combine data from multiple sources
+Dozer can in real-time join data coming from multiple data sources powering advanced use cases. This can be done using the standard JOIN operator in SQL.
+
+* Data API Endpoints
+Note: The SQL you specify in the yaml file does not run in the source database.
+
+
+# 🚀 Quick Start
+You can get started with Dozer in one of two ways:
+
+Dozer Core
```

Comment:
what u mean exactly ? They are not alternative ways. You should create a project and then query data